pkgs/development/python-modules/poetry/default.nix

```diff
@@ -30,22 +30,34 @@
 let
   glob2 = callPackage ./glob2.nix { };
 
+  keyring' = keyring.overridePythonAttrs (old: rec {
+    version = "20.0.1";
+    src= fetchPypi {
+      inherit (old) pname;
+      inherit version;
+      sha256 = "138dbl7931i3gxm1fvvw5b7x5nnrbyg5h9ayvh5x6s8215zzlfwn";
+    };
+  });
+
```

Comment:
this is highly discouraged please try to remove

Reply:
Will add a patch and use substituteAll instead.

Reply:
nvm.